templates/element/sidebar.php

```diff
@@ -24,7 +24,7 @@
                 <?php
                 foreach ($sidebarNavigation as $entry) {
                     if ($entry instanceof MenuItem) {
-                        echo $this->element('menu/item', ['item' => $entry]);
+                        echo $this->element('menu/item', ['item' => $entry, 'itemClass' => 'nav-link']);
```

Comment:
Since `nav-link` is used as default value in the element we can skip setting the same here.